Port ISLANDORA-2124

mghughes opened the pull request:
Drupal 8 port of https://github.com/Islandora/islandora_pdfjs/pull/31

Commit:
Port ISLANDORA-2124

Drupal 8 port of https://github.com/Islandora/islandora_pdfjs/pull/31

qadan commented on islandora_pdfjs.module:
```diff
@@ -74,3 +74,93 @@ function islandora_pdfjs_viewer_callback(array $params, $fedora_object = NULL) {
+  $viewer_id = islandora_get_viewer_id('islandora_book_viewers');
+  if ($viewer_id == 'islandora_pdfjs') {
+    // Grab the names of the appropriate solr fields from the db.
+    $parent_book_field_name = \Drupal::config('islandora_book.settings')->get('islandora_book_parent_book_solr_field', 'RELS_EXT_isMemberOf_uri_ms');
```
Second parameter to `get()` not required

Commit:
Fix config->get calls

Also, I'm pretty sure the page number field is actually called
'islandora_paged_content_sequence_number_field'.
That's in the paged_content module code anyway, so I'm going with that.
